src/CommandingOfficers/Tech.java

```diff
@@ -495,7 +496,7 @@ private XYCoord findDropLocation(GameMap gameMap, Set<XYCoord> priorDrops, boole
       for( XYCoord pdc : options )
       {
         // No unfortunate accidents
-        if( !unitModelToDrop.propulsion.canTraverse(gameMap.getEnvironment(pdc)) )
+        if( !new UnitContext(myCommander, unitModelToDrop).calculateMoveType().canTraverse(gameMap.getEnvironment(pdc)) )
```

Comment:
This is like... so ugly.

Reply:
I could hide it in a function call. Or we could just assume that nobody's ever going to change Tech's movetype such that some terrain is no longer traversable.

Reply:
That is probably a safe assumption, but I also try to avoid thinking any assumption is safe.